CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add module template update
+
+### Changed
+
+- Change `threshold_rules` from required to optional
+- BREAKING CHANGE: Change `amount` from optional to required and to original resource structure
+- Upgrade minimum supported provider version to v4.30 since it `threshold_values` isn't a required variable anymore. For details please see https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#4300-july-25-2022
+
+### Removed
+
+- BREAKING CHANGE: Remove support for Terraform versions before v1.0
+- BREAKING CHANGE: Remove support for Terraform Google Provider versions before v4.30
+
```

Comment:
this hasn't changed... and new minimal version already mentioned. seems to be a copy&paste entry?

```suggestion
```